frontend/app/src/util/AppNavigation.ts

```diff
@@ -38,7 +42,10 @@ export type PageUrlUpdateCallback = (
 ) => void
 export type PageNotFoundCallback = (pageName?: string) => void
 
-export class V1Strategy {
+function getTitle(pageName: string): string {
+  return `${pageName} · Streamlit`
```

Comment:
nit: should we just have this return `Streamlit` when `pageName` is falsy? The separator is a bit weird otherwise